workspaces/ui-v2/src/lib/__tests/diff-helpers/shape-interpretations.test.ts

```diff
@@ -36,7 +31,7 @@ test('a known field is missing.managed', async () => {
   const diff = universe.diffSet.groupedByEndpointAndShapeTrail()[0];
   const result = await shapeDiffPreview(diff, universe.universe);
   commonAssertions(result);
-  await logResult(result.preview);
+  // logResult(result.preview);
```

Comment:
I'm not sure why these were logging, but for future debugging I'm keeping the fn calls here commented out